cmd/pitr/recoverer/recoverer.go

```diff
@@ -33,6 +33,7 @@ type Recoverer struct {
 	recoverFlag    string
 	recoverEndTime time.Time
 	gtid           string
+	verifyTLS	   bool
```

Comment:
can we get it from storage.VerifyTLS ?

Reply:
No. We compile it as a separate binary and use it in [backup image](https://github.com/percona/percona-docker/blob/41929c495fa2ce642930588fe768d09c6c0240d9/percona-xtradb-cluster-8.0-backup/Dockerfile#L8). But we pass `storage.verifyTLS` value to the container [here](https://github.com/percona/percona-xtradb-cluster-operator/blob/d4bb325246a0da319f1b69b161ffb2ccf2218575/pkg/pxc/backup/job.go#L90-L91)